src/stepper/stepper.ts

```diff
@@ -1795,15 +1795,42 @@ function reduceMain(
           firstStatement.type === 'ExpressionStatement' &&
           isIrreducible(firstStatement.expression, context)
         ) {
-          // let stmt
-          // if (otherStatements.length > 0) {
-          paths[0].push('body[0]')
-          paths.push([])
-          const stmt = ast.program(otherStatements as es.Statement[])
-          // } else {
-          //   stmt = ast.expressionStatement(firstStatement.expression)
-          // }
-          return [stmt, context, paths, explain(node)]
+          const [secondStatement] = otherStatements
```

Comment:
should this be
```
const [secondStatement, ...remainingStatements] = otherStatements
```
The pattern matching makes it look like the array has only one element at this point, but it could have more, right?

Aparently, in JS, this program succeeds:
```
const [a] = [1, 2]
display(a) // displays 1
```

Reply:
remainingStatement won't be utilised, so I did not include it in. Are we okay with that?

Reply:
Please refer to my comment offline. In short, I prefer the original implementation without the extra assignment, but for clarity might be good to add a comment explaining that the remaining statements are not used.